Adds function to load window from url in a div or iframe and adds a refresh button

Dravenshorst opened the pull request:
* Adds function to create a window from URL inside iframe or div.
* Also adds a refresh button.

Signed-off-by: Daniel Ravenshorst <daniel.ravenshorst@dealerdirect.nl>

frenck commented:
![bitmoji](https://render.bitstrips.com/v2/cpanel/b60cb965-cc0a-4365-8d1f-8d56e9255948-82100c49-8c0a-4ae4-9b04-3a6d3eac9cc2-v1.png?transparent=1&palette=1&width=246)

rlamana commented:
@Dravenshorst I still have mixed feelings about this PR. The idea of the library is to keep it as light and simple as possible and just focus on window management, not on the content of the windows. I see it as just a wrapper around content, no matter if it is a single DOM element, a complex DOM tree or an **iframe**. That is up to the developer of the wrapped "app"/content.

What counter arguments do you have to convince me to include it as part of ventus? ;)

Dravenshorst commented:
@rlamana I understand your concerns.
The main goal of this PR is to add an refresh button. 
So I have changed this PR to listen to the reload event. If the reload event is defined it will show a refresh button. The functionality of the button is up to the user an can be defined in the event callback. 
I have added an working example of how I am using it in `examples/simple`.

Can you have a look at it again.
Thanks.

rlamana commented:
@Dravenshorst A more generic approach (align with the goal of this library) would be to provide a way to customize the content of the title bar. Maybe we could provide an flexible area, between the title and the buttons where the developer can attach/append a DOM element and extend the functionality of the title bar. 

Your solution seems to me too coupled to the code that solves your iframe requirement, don't you think?

frenck commented:
Closing without saying anything! Best 👍 
Thanks, m8!

rlamana commented:
@frenck  Sorry I assumed this PR was abandoned. It's been almost 2 years and I got no response to the conversation I started. Also, there are merge conflicts that haven't been fixed. I'm more than glad to reopen it if someone is going to continue with it.

frenck commented:
I'm not involved in the company that used it anymore, so I don't mind. I just received a closing notification without any comment on this PR.

In general, I think that is not nice. At least have the decency to leave a message with something like: "Yooo m8, this PR looks stale, will close it for now. Feel free to re-open when you are ready to work on it again" or similar.
